elcid/models.py

```diff
@@ -319,13 +334,6 @@ class Allergies(omodels.Allergies, ExternallySourcedModel):
     allergy_start_datetime = models.DateTimeField(null=True, blank=True)
     no_allergies = models.BooleanField(default=False)
 
-    @classmethod
-    def get_fieldnames_to_serialize(cls):
-        # Looks like we won't get these from the prescribing system
```

Comment:
The method name is _get_fieldnames_to_serialize also I'm not sure this would remove any additional fields. I'm guessing it was a typo